composer.json

```diff
@@ -15,21 +15,21 @@
         }
     ],
     "require": {
-        "php": "^8.1.0",
+        "php": "^8.2.0",
         "nyholm/psr7": "^1.8.1",
         "openai-php/client": "^0.8.0",
         "psr/http-client": "^1.0.3",
         "psr/http-factory": "^1.0.2",
-        "symfony/config": "^5.4.21|^6.3.8",
-        "symfony/dependency-injection": "^5.4.21|^6.3.8",
-        "symfony/http-client": "^5.4.21|^6.3.8",
-        "symfony/http-kernel": "^5.4.21|^6.3.8"
+        "symfony/config": "^7.0.0",
```

Comment:
There is no reason to change the minimum values.
`^5.4|^6.3|^7.0` should be the used as all this versions are compatibles.